src/Sdk/Subscriptions/types.ts

```diff
@@ -0,0 +1,56 @@
+import { Contact, ContactType, UnsubscribeReason, UploadcareImage } from '../../types';
+
+interface Person {
+    contact_type: ContactType.PERSON;
+    first_name?: Contact['first_name'];
+    last_name?: Contact['last_name'];
+    gender?: Contact['gender'];
+    function_name?: Contact['function_name'];
+    organisations?: string[];
```

Comment:
None of these attributes are required - is this correct?

Because previously comments in this file stated:

> Required only when contact_type === 'person'

Reply:
Great catch! But it was a wrong doc before. 😬  These parameters are actually not required.